storage/inspector/src/main.rs

```diff
@@ -0,0 +1,101 @@
+use libradb::LibraDB;
+use std::path::Path;
+use transaction_builder::get_transaction_name;
+
+// Print out latest information stored in the DB.
+fn print_head(db: &LibraDB) {
+    let version = db
+        .get_latest_version()
+        .expect("get_latest_version throw error");
+
+    println!("Latest Version: {}", version);
+
+    let si = db
+        .get_startup_info()
+        .expect("Can't get startup info")
+        .expect("StartupInfo is empty, database is empty.");
+
+    println!(
+        "The latest ledger info: {}",
+        si.latest_ledger_info.ledger_info()
+    );
+
+    println!("Signatures: {:?}", si.latest_ledger_info.signatures());
+
+    println!(
+        "Current Validator Set: {}",
+        si.latest_validator_set
+            .expect("Unable to determine validator set, DB incorrect")
+    );
+
+    let tx = db
+        .get_transaction_with_proof(version, version, false)
+        .expect("Unable to load latest TXN");
+    println!(
+        "Current Transaction: {}",
+        tx.transaction.format_for_client(get_transaction_name)
+    );
+}
+
+// Some naive implementation to verify local storage consistency by iterating over information
+// ideally should be implemented inside Storage layer directly.
+fn verify_consistency(db: &LibraDB) {
+    let si = db
+        .get_startup_info()
+        .expect("Can't get startup info")
+        .expect("StartupInfo is empty, database is empty.");
+    let ledger_info = si.latest_ledger_info.ledger_info();
+
+    let latest_version = ledger_info.version();
+
+    for n in (1..latest_version).rev() {
+        // This will force DB to scan TX trees and event trees
+        let tx = db
+            .get_transaction_with_proof(n, latest_version, true)
+            .expect("Unable to load transaction");
+
+        // This will force DB to scan AccountState tree, not all account though
+        db.get_account_state_with_proof_by_version(Default::default(), tx.version)
```

Comment:
Not sure if I understand it correctly, it appears to me that we are trying to validate we can read back Txn history: 1) going through every version in ledger_version, and 2) verifying we can at least read back the state of one account associated with that version used in txn.